examples/weather/pangu_weather/train_pangu_lite_era5.py

```diff
@@ -0,0 +1,331 @@
+                    5,
+                    -1,
+                    data[0]["invar"].shape[2],
+                    data[0]["invar"].shape[3],
+                )
+            )
+        )
+        outvar_surface = data[0]["outvar"].cpu().detach()[:, :, :4, :, :]
+        outvar_upper_air = (
+            data[0]["outvar"]
+            .cpu()
+            .detach()[:, :, 4:, :, :]
+            .reshape(
+                (
+                    data[0]["outvar"].shape[0],
+                    data[0]["outvar"].shape[1],
+                    5,
+                    -1,
+                    data[0]["outvar"].shape[3],
+                    data[0]["outvar"].shape[4],
+                )
+            )
+        )
+        predvar_surface = torch.zeros_like(outvar_surface)
+        predvar_upper_air = torch.zeros_like(outvar_upper_air)
+
+        for t in range(outvar_surface.shape[1]):
+            output_surface, output_upper_air = eval_step(
+                pangu_model, invar_surface, surface_mask, invar_upper_air
+            )
+            invar_surface.copy_(output_surface)
+            invar_upper_air.copy_(output_upper_air)
+            predvar_surface[:, t] = output_surface.detach().cpu()
+            predvar_upper_air[:, t] = output_upper_air.detach().cpu()
+
+        num_elements_surface = torch.prod(torch.Tensor(list(predvar_surface.shape[1:])))
+        num_elements_upper_air = torch.prod(
+            torch.Tensor(list(predvar_upper_air.shape[1:]))
+        )
+        loss_epoch += (
+            torch.sum(torch.pow(predvar_surface - outvar_surface, 2))
+            + torch.sum(torch.pow(predvar_upper_air - outvar_upper_air, 2))
+        ) / (num_elements_surface + num_elements_upper_air)
+
+        num_examples += predvar_surface.shape[0]
+
+    pangu_model.train()
+    return loss_epoch / num_examples
+
+
+@hydra.main(version_base="1.2", config_path="conf", config_name="config")
+def main(cfg: DictConfig) -> None:
+    DistributedManager.initialize()
+    dist = DistributedManager()
+
+    # Initialize loggers
+    # initialize_wandb(
+    #     project="Modulus-Launch-Dev",
+    #     entity="Modulus",
+    #     name="FourCastNet-Training",
+    #     group="FCN-DDP-Group",
+    # )
+    initialize_mlflow(
+        experiment_name="Modulus-Launch-Dev",
+        experiment_desc="Modulus launch development",
+        run_name="Pangu-Training",
+        run_desc="Pangu ERA5 Training",
+        user_name="Modulus User",
+        mode="offline",
+    )
+    LaunchLogger.initialize(use_mlflow=cfg.use_mlflow)  # Modulus launch logger
+    logger = PythonLogger("main")  # General python logger
+
+    no_channals_pangu = 4 + 5 * 13
+    datapipe = ERA5HDF5Datapipe(
+        data_dir="/data/train/",
+        stats_dir="/data/stats/",
+        channels=[i for i in range(no_channals_pangu)],
+        num_samples_per_year=cfg.num_samples_per_year_train,
+        batch_size=1,
+        patch_size=(1, 1),
+        num_workers=8,
```

Comment:
Same comment, can these be configurable.

Reply:
Moved to conf/config.yaml